products/chrome.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Chrome
+category: app
+tags:
+  - google
+  - web-browser
+iconSlug: googlechrome
+permalink: /chrome
+alternate_urls:
+-   /google-chrome
+versionCommand: google-chrome --version
+releasePolicyLink: https://blog.chromium.org/2021/03/speeding-up-release-cycle.html
+changelogTemplate: "https://developer.chrome.com/release-notes/__LATEST__"
+releaseColumn: false
+
+identifiers:
+- repology: google-chrome
+- cpe: cpe:2.3:a:google:chrome
+
+releases:
+-   releaseCycle: "137"
+    releaseDate: 2025-05-27
+    eol: false
+-   releaseCycle: "136"
+    releaseDate: 2025-04-29
+    eol: 2025-05-27
+-   releaseCycle: "135"
+    releaseDate: 2025-04-01
+    eol: 2025-04-29
+-   releaseCycle: "134"
+    releaseDate: 2025-03-04
+    eol: 2025-04-01
+-   releaseCycle: "133"
+    releaseDate: 2025-02-04
+    eol: 2025-03-04
+-   releaseCycle: "132"
+    releaseDate: 2025-01-14
+    eol: 2025-02-04
+-   releaseCycle: "131"
+    releaseDate: 2024-11-12
+    eol: 2025-01-14
+-   releaseCycle: "130"
+    releaseDate: 2024-10-15
+    eol: 2024-11-12
+-   releaseCycle: "129"
+    releaseDate: 2024-09-17
+    eol: 2024-10-15
+-   releaseCycle: "128"
+    releaseDate: 2024-08-20
+    eol: 2024-09-17
+-   releaseCycle: "127"
+    releaseDate: 2024-07-23
+    eol: 2024-08-20
+-   releaseCycle: "126"
+    releaseDate: 2024-06-11
+    eol: 2024-07-23
+-   releaseCycle: "125"
+    releaseDate: 2024-05-14
+    eol: 2024-06-11
+
+---
+
+> [Google Chrome](https://www.google.com/chrome/) is a web browser developed by Google.
+> Chrome is available for
+> [Windows](https://support.google.com/chrome/answer/95346),
+> [macOS](https://support.google.com/chrome/answer/95346),
+> [Android](https://support.google.com/chrome/answer/95346),
+> [iPhone & iPad](https://support.google.com/chrome/answer/95346),
+> [Linux](https://support.google.com/chrome/answer/95346), and
+> [ChromeOS](https://support.google.com/chrome/answer/95346).
+
+Chrome is released every [4 weeks](https://blog.chromium.org/2021/03/speeding-up-release-cycle.html).
```

Comment:
It would be nice to explain the different [release channels](https://developer.chrome.com/docs/web-platform/chrome-release-channels).